joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/macros/MacroHandlingTests.scala

```diff
@@ -317,8 +317,8 @@ class MacroHandlingTests extends C2CpgSuite {
           |}
           |""".stripMargin)
       cpg.local.count(l => l._astIn.isEmpty) shouldBe 0
-      cpg.local.count(l => l._astIn.size == 1) shouldBe 4
       cpg.local.count(l => l._astIn.size > 1) shouldBe 0
+      cpg.local.count(l => l._astIn.size == 1) shouldBe 3
```

Comment:
what has changed here? One local has disappeared?

Reply:
There was an issue with lambdas behind macros in polarphp that triggered my investigation on Fabs Macro call handling. There I fixed an issues. This additional local should have never been there in first place.